.gitignore

```diff
@@ -1,3 +1,7 @@
+# Sphinx documentation
+docs/_build/
+docs/api/
```

Comment:
My thinking is that the api docs should not be in source control. They're based on the source code and should be generated at docs-build-time.

Reply:
:100: